src/Syncromatics.Clients.WaySine/PolledDateTimeConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Syncromatics.Clients.WaySine
+{
+    public class PolledDateTimeConverter : DateTimeConverterBase
+    {
+        public override bool CanWrite => false;
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            if (reader.TokenType != JsonToken.Integer) return null;
+            var ticks = (long)reader.Value;
+            return new DateTime(1970, 1, 1).AddMilliseconds(ticks);
```

Comment:
I recommend using DateTimeOffset.FromUnixTimeMilliseconds()

Reply:
That's much better.  I'll add it.  Thanks!